File: app/src/main/java/com/sp/base/database/DBMigration.java
package com.sp.base.database;

import androidx.room.migration.Migration;
import androidx.sqlite.db.SupportSQLiteDatabase;

import org.jetbrains.annotations.NotNull;

public class DBMigration {

    public static final Migration MIGRATION_2_3 = new Migration(5, 6) {
        @Override
        public void migrate(@NotNull SupportSQLiteDatabase database) {
            /*database.execSQL("ALTER TABLE virtual_card " + " ADD COLUMN custId TEXT");*/
        }
    };
}

File: app/src/main/java/com/baseapplication/database/DBMigration.kt
package com.baseapplication.database

import androidx.room.migration.Migration
import androidx.sqlite.db.SupportSQLiteDatabase

object DBMigration {
    val MIGRATION_2_3: Migration = object : Migration(5, 6) {
        public override fun migrate(database: SupportSQLiteDatabase) {
            /*database.execSQL("ALTER TABLE virtual_card " + " ADD COLUMN custId TEXT");*/
        }
    }
}
